video-library/src/components/History/HistoryComp.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import "./History.css"
+import SideBar from "../SideBar/SideBar";
+import HistoryVideos from './HistoryVideos';
+function HistoryComp() {
```

Comment:
I have a suggestion, file and component name "HistoryComp" can be changed to a more precise word